### ISSUE TITLE
%pistol-extra% substitution not working.

### ISSUE THREAD

**Issue by meck** [doronbehar/pistol#56]
I just tried the new functionally for image previewing discovered a small issue.
```
image/* chafa -s %pistol-extra0%x%pistol-extra1% %pistol-filename%
```
does not work it produces nothing  for `%pistol-extra0%x%pistol-extra1%` but if i add some spaces:
`%pistol-extra0% x %pistol-extra1%` it produces the expected output eg. `10 x 20`.

**Comment by doronbehar** [comment]
cc @lucas-mior .

**Comment by lucas-mior** [comment]
That's right, pistol expects to these arguments to be separated from others.
I don't see a simple way to fix this, but until then you should be able to wrap your chafa command inside a script.

**Comment by doronbehar** [comment]
Maybe use `sh`? https://github.com/doronbehar/pistol/blob/master/README.adoc#quoting-and-shell-piping

**Comment by meck** [comment]
Yes, the following roundabout  in the configuration seems to work:

```
image/* sh: size=" %pistol-extra0% x %pistol-extra1% ";size="$(echo -e "${size}" | tr -d '[:space:]')"; chafa -s "${size}" %pistol-filename%
```

**Comment by doronbehar** [comment]
This is still an issue worth solving, that workaround is too ugly. Something is wrong with:

https://github.com/doronbehar/pistol/blob/2e645a020d998ef68d0c824f878a3e95ed0f852a/previewer.go#L170

It doesn't replace the strings if they are not separated with space..
